src/Symfony/Component/Translation/Bridge/Crowdin/CrowdinProvider.php

```diff
@@ -330,6 +332,26 @@ private function downloadSourceFile(int $fileId): ResponseInterface
         return $this->client->request('GET', sprintf('files/%d/download', $fileId));
     }
 
+    private function listLanguages(): array
+    {
+        /**
+         * @see https://developer.crowdin.com/api/v2/#operation/api.languages.getMany (Crowdin API)
+         * @see https://developer.crowdin.com/enterprise/api/v2/#operation/api.languages.getMany (Crowdin Enterprise API)
+         */
+        // cannot query by locales, we need to fetch all and filter out the relevant ones
+        $response = $this->client->request('GET', '../../languages', [
```

Comment:
let's make this simpler and inline the query string: `request('GET', '../../languages?limit=500');`